Source/WebCore/animation/ViewTimeline.cpp

```diff
@@ -305,28 +295,31 @@ ScrollTimeline::Data ViewTimeline::computeTimelineData(const TimelineRange& rang
         case SingleTimelineRange::Name::Entry:
         case SingleTimelineRange::Name::EntryCrossing:
             return subjectOffset - scrollContainerSize - subjectSize;
-        default:
-            ASSERT_NOT_REACHED();
+        case SingleTimelineRange::Name::Omitted:
             return 0.f;
         }
+        ASSERT_NOT_REACHED();
+        return 0.f;
     };
 
     auto rangeStart = computeRangeStart() + insetEnd.value();
     auto rangeEnd = computeRangeEnd() - insetStart.value();
 
-    return { currentScrollOffset, rangeStart + ScrollTimeline::floatValueForOffset(range.start.offset, rangeEnd - rangeStart), rangeEnd - ScrollTimeline::floatValueForOffset(range.end.offset, rangeEnd - rangeStart) };
+    return { currentScrollOffset, rangeStart + ScrollTimeline::floatValueForOffset(range.start.offset, rangeEnd - rangeStart), rangeStart + ScrollTimeline::floatValueForOffset(range.end.offset, rangeEnd - rangeStart) };
```

Comment:
Could be easier to read split into multiple lines too

Reply:
Fixed.